source/MaterialXCore/Value.cpp

```diff
@@ -14,11 +14,11 @@
 MATERIALX_NAMESPACE_BEGIN
 
 Value::CreatorMap Value::_creatorMap;
-Value::FloatFormat Value::_floatFormat = Value::FloatFormatDefault;
-int Value::_floatPrecision = 6;
 
 namespace
 {
+thread_local Value::FloatFormat _floatFormat = Value::FloatFormatDefault;
+thread_local int _floatPrecision = 6;
```

Comment:
Previously, these were `static` data members of the `Value` class, mutated during codegen by `ScopedFloatFormatting`.

Making them `thread_local` fixed the data race, but they couldn't remain members because the class is marked as `__declspec(dllexport)`  and TLS variables can't be exported. So, moving them to file scope.

I'm not quite sure what the naming convention for file-scoped variables is.